.github/workflows/CI.yml

```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   pull_request:
     branches: [master, nightly]
-    types: [opened, synchronize, edited, reopened]
+    types: [opened, synchronize, reopened]
   push:
     branches: [master]
```

Comment:
Add `nightly` branch push event.

Will require special consideration for standard binaries. Perhaps we can publish a `pre-release`?

Reply:
Publishing pre-release versions will be a separate issue to resolve.